dotcom-rendering/src/web/experiments/tests/sign-in-gate-mandatory-long-testrun.ts

```diff
@@ -168,21 +17,20 @@ export const signInGateMandatoryLongTestRunEu: ABTest = {
 	successMeasure: 'Users sign in or create a Guardian account',
 	audienceCriteria:
 		'Global, 3rd article of the day, lower priority than consent banner, simple articles (not gallery, live etc.), not signed in, not shown after dismiss, not on help, info sections etc. Exclude iOS 9 and guardian-live-australia. Suppresses other banners, and appears over epics',
-	dataLinkNames: 'SignInGateMandatoryLongTestRunEu',
+	dataLinkNames: 'SignInGateMandatoryLongTestRunUk',
```

Comment:
Should this be `SignInGateMandatoryLongTestRunBucketingRun`?

Reply:
yes! Thank you for spotting!